packages/plugins/rum/src/sourcemaps/sender.ts

```diff
@@ -96,7 +96,9 @@ export const upload = async (
         return { errors, warnings };
     }
 
-    const queue = new PQueue({ concurrency: options.maxConcurrency });
+    // @ts-expect-error PQueue's default isn't typed.
+    const Queue = PQueue.default ? PQueue.default : PQueue;
+    const queue = new Queue({ concurrency: options.maxConcurrency });
```

Comment:
This is the usual mess of highly opinionated Sindre Sorhus' packages.

Reply:
bummer :(

```
exports.default = PQueue;
```